addons/mrp/views/product_views.xml

```diff
@@ -132,6 +165,35 @@
                         </div>
                     </button>
                 </xpath>
+                <xpath expr="//button[@name='action_open_quants']" position="after">
+                    <button class="oe_stat_button"
```

Comment:
As the buttons should be called `Available` for both storable and kit consumable product, I would leave the button in `stock` and simply set show_on_hand_qty_status_button to True or False in mrp 

Same comment about the forecasted button

Reply:
I don't think so. These buttons are only for kits product (any product with a kits bom, no matter it's storable or consumable).